bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,935 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="souliss"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<bridge-type id="gateway">
+		<label>Gateway</label>
+		<description>Represents a Souliss Gateway.</description>
+
+		<config-description>
+			<parameter name="GATEWAY_IP_ADDRESS" type="text" required="true">
```

Comment:
GATEWAY_IP_ADDRESS
Should be lowerCamelCase, same applies to all channels and parameters so I wont flag all the cases.

see
https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Reply:
changed